meteor/client/helpers/google_maps.js

```diff
@@ -94,7 +105,7 @@ gmaps = {
 
       var googleLoc = new google.maps.LatLng(latLng.lat, latLng.lng);
       gmaps.selfMarker.setPosition(googleLoc);
-    
+
     } else {
 
       if (navigator.geolocation) navigator.geolocation.getCurrentPosition(function(pos) {

```

Comment:
@drenfr01 it means the same thing as if it has brackets - it only works for single line statements though. The reason it was added was because I changed the getCurrentLocation to call the callback with the standard (error, result) arguments. I stumbled across a bunch of these errors while on the train (with the geolocation). They should now be handled.